examples/README.md

```diff
@@ -12,6 +12,32 @@
                       limitations under the License.
 
 # Examples of Enceladus usage
+## Templates for _INFO files
+#### Templates which illustrate the way to write _INFO files, one with the minimally necessary fields and one with the recommended fields to add
```

Comment:
This can be ignored but by the Markdown style guide, every level of heading need to be surrounded by a new line. In the case of two headings after each other, one new line is enough.

Reply:
Now that I focus on it, why is everything a heading?

Reply:
Oh, I missed that. Good point.

Reply:
Line 16 doesn't need to be a heading indeed